lib/commands/addToMailchimp.js

```diff
@@ -0,0 +1,20 @@
+const Mailchimp = require('mailchimp-api-v3')
+
+const mailchimp = new Mailchimp(process.env.MAILCHIMP_API_KEY);
+
+module.exports.postToMailchimp = function(userInformation) {
```

Comment:
entirely style related comment, feel free to ignore, but i think it's generally better to do something like:

```
const postToMailchimp = function(...) {
...
}

module.exports = {
  postToMailchimp,
};
```